src/documentation.ts

```diff
@@ -141,6 +145,28 @@ export default class Documentation {
         return details;
     }
 
+    private serializeClass(symbol: ts.Symbol, fileName: string) {
+        const details: IInterfaceEntry = {
+            documentation: ts.displayPartsToString(symbol.getDocumentationComment()),
+            fileName,
+            name: symbol.getName(),
+            tags: this.getJsDocTags(symbol),
+            type: "class",
+        };
+
+        const classNode = symbol.declarations[0] as ts.ClassDeclaration;
+        if (classNode.heritageClauses != null) {
+            details.extends = classNode.heritageClauses[0].types.map((type) => type.getText());
+        }
+
+        // Get the props signatures
```

Comment:
"member signatures"

big question: how similar are members to props? are there things that members can do that props can't? what about `private/public`?